internal/content/repository/mysql/mysql_content_test.go

```diff
@@ -26,7 +26,7 @@ func TestGetAll(t *testing.T) {
 
 	query := `SELECT id,title, updated_at, created_at FROM contents ORDER BY created_at DESC LIMIT \?,\?`
 	mock.ExpectQuery(query).WillReturnRows(rows)
-	c := tagMysqlRepo.InitMysqlRepository(db)
+	c := mysqlrepo.InitMysqlRepository(db)
```

Comment:
Minor: I'd be nice to rename this method
```suggestion
	c := mysqlrepo.Init(db)
```
But you can cover it in another PR.

Reply:
ooh yeah.